### ISSUE TITLE
inconsistent naming and no longer exists icon

### ISSUE THREAD

**Issue by calvintam236** [galonsos/line_icons#24]
For example, per [Icons8](https://icons8.com/line-awesome):
* inconsistent naming
  * `store-alt` -> `alternateStore`
  * `hospital-alt` -> `hospitalAlt`
* no longer exists
  * `buildingAlt` (`building-1` no longer exists)

There are more than I shown here.

I think latest version of this package should always has the current icons set, and icon(s) removed should not be supported by adding `@Deprecated` tag.

**Comment by mathtasatlime** [comment]
This is also a hugh problem for me! Pls fix it!

**Comment by lukehutch** [comment]
I agree it would be much better to exactly follow the naming given on the Line Awesome website. Putting `alternate` at the beginning rather than using `-alt` as a suffix is an unnecessary change.

**Comment by galonsos** [comment]
Three points here:

* Naming has to follow Dart/Flutter conventions (`whatever-alt` becomes `whateverAlt`).
* The naming was already given automatically, so inconsistency doesn't come from me, but from the metadata included in the font itself.
* The package is not regularly mantained, as you may notice :). I made it for my own projects and published it, but that's all. That means I'm not closely watching changes on the original Line Awesome site and the package doesn't reflect the latest version of Line Awesome icon set.

That said, I'll take into account your suggestiones for any future update. Thank you, guys!

**Comment by lukehutch** [comment]
@galonsos But the point is you didn't follow the pattern in the original library: `whatever-alt` is not `whateverAlt`, you made it into `alternateWhatever`...

`whateverAlt` is fine, but `alternateWhatever` makes it very hard to find icons via autocomplete sometimes, because that name doesn't even start with the same letter as the official icon name.
